packages/stream_chat/CHANGELOG.md

```diff
@@ -1,3 +1,28 @@
+## 3.4.0
+
+🐞 Fixed
+
+- [[#857]](https://github.com/GetStream/stream-chat-flutter/issues/857) Channel now listens for member ban/unban and
+  updates the channel state with the latest data.
+- [[#748]](https://github.com/GetStream/stream-chat-flutter/issues/748) `Message.user` are now also included while saving users in persistence.
+- [[#871]](https://github.com/GetStream/stream-chat-flutter/issues/871) Fixed thread message deletion.
+- [[#846]](https://github.com/GetStream/stream-chat-flutter/issues/846) Fixed `message.ownReactions` getting truncated when receiving a reaction event.
+- Add check for invalid image URLs
+- Fix `channelState.pinnedMessagesStream` getting reset to `0` after a channel update.
+
+🔄 Changed
+
+- `client.location` is now deprecated in favor of the
+  new [edge server](https://getstream.io/blog/chat-edge-infrastructure) and will be removed in v4.0.0.
+- `channel.banUser`, `channel.unbanUser` is now deprecated in favor of the new `channel.banMember`
+  and `channel.unbanMember` and will be removed in v4.0.0.
+
```

Comment:
- Added `banExpires` property of type `DateTime` on the `Member`, `OwnUser`, and `User` models.

Reply:
```suggestion
- Added banExpires property of type DateTime on the Member, OwnUser, and User models.
```